### PR TITLE
Update pom.xml

### LINKED ISSUE #44: The maven plugin isn't at the latest version
<artifactId>liberty-maven-plugin</artifactId>
                <version>2.6.1</version>

### PR DESCRIPTION
**Pull request by gkwan-ibm**
fix #44 and #40

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ include::finish/src/resources/artists.json[]
 ----
 
 You will implement an AngularJS client that consumes this JSON and displays its contents at the
-following URL: http://localhost:9080[http://localhost:9080^].
+following URL: \http://localhost:9080/index.html.
```
Comment (andrewdes) [inline README.adoc]: Is the forward slash at the start of the URL needed?

Comment (andrewdes) [reply README.adoc]: Link should be in `backticks`

File: README.adoc [hunk 2]
```diff
@@ -80,12 +80,6 @@ After you start the server, you can find your artist JSON at the following URL:
 Any local changes to your JavaScript and HTML are picked up automatically. After you start the Open
 Liberty server, you do not need to restart it.
 
-When you are done checking out the services, stop the Open Liberty server by running the following command:
-[role='command']
-```
```
Comment (andrewdes) [inline README.adoc]: Why is this being removed?

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: I don't want reader to run the url at there because server is not installed and started

**Pull request merged**